bolt-design-system/bolt
# Preparing your environment update for PHP 7.3 users

## Summary

Update documentation for PHP 7.3

## Details

Update documentation for PHP 7.3

## How to test

Run code locally and go to: `Getting Started Guide > Preparing your environment` then scrool to section: `4. Install PHP and PHP Dependencies` and check `Attention PHP 7.3 users` section.
